core/ulog/log.go

```diff
@@ -104,96 +71,29 @@ type Log interface {
 	DFatal(message string, keyvals ...interface{})
 }
 
-var development = strings.Contains(config.GetEnvironment(), "development")
-
-var std = defaultLogger()
+var _std = defaultLogger()
 
 func defaultLogger() Log {
 	return &baselogger{
-		log:        zap.New(zap.NewJSONEncoder()),
-		initFields: nil,
+		log: zap.New(zap.NewJSONEncoder()),
 	}
 }
 
 // Logger returns the package-level logger configured for the service
-func Logger(fields ...interface{}) Log {
+func Logger() Log {
```

Comment:
I am keeping this method for now, even though ulog can be created using NewBuilder.Build(). This method is used in most of the places in fx, and warrants a separate PR. I will add a TODO here.